README.md

```diff
@@ -17,6 +17,10 @@ Feel free to **Star** and **Watch** it, but watch out for falling debris.
 
 Feedback and contributions are always welcome and appreciated.
 
+### Gitter Chat Room
```

Comment:
No need for a headline.

See this as a reference: https://raw.githubusercontent.com/DAVFoundation/missioncontrol/master/README.md